adcm_client/objects.py

```diff
@@ -431,9 +431,10 @@ def service_add(self, **args) -> "Service":
             data = self._subcall("service", "create", prototype_id=proto.id)
             return self._subobject(Service, service_id=data['id'])
 
-#    @allure_step("Remove service to cluster")
-#    def service_delete(self, service):
-#        self._subcall("service", "delete", service_id=service.id)
+    @version_compatibility_check('2020.05.13.00')
+    def service_delete(self, service: "Service"):
```

Comment:
Please add one more method. It would be nice to have delete() method on class Service.

Reply:
Method delete() is already implemented in the parent class BaseApiObject